recipes/dns.rb

```diff
@@ -1,5 +1,8 @@
 include_recipe 'dnsmasq::default'
-include_recipe 'dnsmasq::manage_hostsfile'
+
+if(node[:dnsmasq][:enable_hostsfile])
+  include_recipe 'dnsmasq::manage_hostsfile'
+end
```

Comment:
Do this with an inline conditional:

```ruby
include_recipe 'dnsmasq::manage_hostsfile' if node['dnsmasq']['enable_hostsfile']
```

This makes it easier to follow the recipe without having to jump through a conditional mentally.